test/controllers/videos_controller_test.rb

```diff
@@ -0,0 +1,37 @@
+require "test_helper"
+
+describe VideosController do
+  VIDEO_FIELDS = ["id", "title", "release_date", "available_inventory"].sort
+
+  it "must get index" do
+    get videos_path
+
+    must_respond_with :success
+    expect(response.header['Content-Type']).must_include 'json'
+  end
+
+  it "will return all the proper fields for a list of videos" do
+    get videos_path
+
+    body = JSON.parse(response.body)
+
+    expect(body).must_be_instance_of Array
+
+    body.each do |video|
+      expect(video).must_be_instance_of Hash
+      expect(video.keys.sort).must_equal VIDEO_FIELDS
+    end
+  end
+
+  it "returns an empty array if no videos exist" do
+    Video.destroy_all
+
+    get videos_path
+
```

Comment:
Should we add this line here to check that it responds with status 200 when there are no videos? 

```suggestion
must_respond_with :ok

```

Reply:
Agree! I have it in line 33 but am not sure the difference between using `:success` and `:ok`. It seems to both pass the test?

Reply:
Ahh I didn't look close enough, sorry! I think they both check for 200 so this looks great!